userspace/falco/configuration.cpp

```diff
@@ -258,6 +260,18 @@ void falco_configuration::load_yaml(const std::string& config_name, const yaml_h
 	}
 
 	m_buffered_outputs = config.get_scalar<bool>("buffered_outputs", false);
+	m_outputs_queue_capacity = config.get_scalar<size_t>("outputs_queue.capacity", DEFAULT_OUTPUTS_QUEUE_CAPACITY_UNBOUNDED_MAX_LONG_VALUE);
```

Comment:
can we use an `int64_t` instead of `size_t`? in this way, the `DEFAULT_OUTPUTS_QUEUE_CAPACITY` could be simply be `-1`

Reply:
We can, at the same time always preferring to stay close to the framework, here TBB, and they use `size_t`. Let me know if its a [nit] or if we significantly gain from this change, then I'll do it.